apps/meteor/app/livechat/imports/server/rest/departments.ts

```diff
@@ -147,12 +149,87 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'livechat/department/archived',
```

Comment:
Since this is a get, rename to `livechat/departments/archived`.

BTW, do we need an endpoint for this? Not sure if we can have this as part of current `find` endpoint with just an extra param (so it can search archived deps too)